docs/findings/el9.md

```diff
@@ -0,0 +1,332 @@
+
+# Configure tmux to lock session after inactivity
+
+**Invalid Finding:**
+
+The configuration-automation within `watchmaker` will configure the `tmux` service per the STIGs. If this finding pops up, it will be necessary to ensure that:
+
+* The associated `watchmaker` state (`.../el9/RuleById/medium/content_rule_configure_tmux_lock_after_time`) actually ran _and_ ran to successful completion
+* The `watchmaker`-set value is the same as the site's prescribed-value
+
+# Configure the tmux Lock Command
+
+**Invalid Finding:**
+
+The configuration-automation within `watchmaker` will configure the `tmux` service per the STIGs. If this finding pops up, it will be necessary to ensure that:
+
+* The associated `watchmaker` state (`.../el9/RuleById/medium/content_rule_configure_tmux_lock_command`) actually ran _and_ ran to successful completion
+* The `watchmaker`-set value is the same as the site's prescribed-value
+
+# Only Authorized Local User Accounts Exist on Operating System
+
+**Expected Finding:**
+
+"Authorized Local User Accounts" is a wholly site-specific determination. As some scanners note in their report-output:
+
+```
+Automatic remediation of this control is not available due to the unique requirements of each system
+```
+
+As a result, most scanners will emit this in their findings-reports as an indication to the accreditor that a manual check of the system's local users conform to site-local policies
+
+# Set the UEFI Boot Loader Admin Username to a Non-Default Value
+
+**Invalid Finding:**
+
+By default, `watchmaker` will attempt to set a custom superuser name for the UEFI bootloader. If the `watchmaker` user does not set the `ash-linux:lookup:grub-user` Pillar parameter to a site-custom value, a default value will be used. Currently, this default value is `grubuser`.
+
+```{eval-rst}
+.. warning::
+   It is `highly` recommended that a site-specific value be set for the
+   ``ash-linux:lookup:grub-user`` Pillar parameter. While failing to do so will
+   not result in a scan-finding, it will mean that anyone that has read this
+   document will know your servers' bootloader superuser name
+```
+
+# Ensure Logs Sent To Remote Host
+
+**Expected Finding:**
+
+"Ensure Logs Sent To Remote Host" is a wholly site-specific determination. While most scanners will look for whether log-offloading via `rsyslog` has been set up, this scan-criteria is generally not valid:
+
+* Many sites use tools _other than_ `rsyslog` to handle log-offloading (Splunk, FluentBit, CSP-specific log-agents have all been used by various organizations that use watchmaker to harden their systems
+* Even sites that _do_ use `rsyslog` to handle log-offloading, the scanners frequently look only for the log-destination `logcollector` - or similarly-generic destination-name - rather than the hostname, FQDN or IP address of the log-collection server
+
+It will be up to the system accreditor to know the site-specific implementation-requirements and validate accordingly
```

Comment:
"accreditation" is dead. "assessment" is the correct term nowadays.

```suggestion
It will be up to the system assessor to know the site-specific implementation-requirements and validate accordingly
```